pkg/infra/teams/team1/social_capital.go

```diff
@@ -141,3 +143,85 @@ func (s *SocialAgent) updateSocialCapital(self agent.BaseAgent, fightDecisions d
 		s.socialCapital[key] = boundArray(s.socialCapital[key])
 	}
 }
+
+/**
+ * Tell other trusted agents above a threshold T about the A agents they admire the most,
+ * and the H agents they hate the most.
+ * Currently messages may be sent to recipients praising or denouncing the recipient
+ */
+func (r *SocialAgent) sendGossip(agent agent.BaseAgent) {
+	selfID := agent.Name()
```

Comment:
think you mean to use agent.ID() and not agent.Name()?

Reply:
I'm actually not sure what the difference is. Maybe I should be using ID, but then where does Name come from? @neeldug do you happen to know?

Reply:
Name was a human readable representation that is optionally present, afaik it was added long ago mainly for logging reasons though.